KOGITO-5170 - Allow using environment variables to define connection … (#3574)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/release-notes/chap-kogito-release-notes.adoc

The generated SVG is named to {processFileName}-svg.svg. Also, the process code generation searches the export and if the file exists, the file is copied and renamed to `META-INF/processSVG/{processId}.svg`, which allows the process SVG addon to access the file directly.

==== PostgreSQL persistence addon configuration 

> It will now be possible to configure the connection with PostgreSQL using environment variables. For instance, using `PGUSER` and `PGPASSWORD` to specify the username and password. For more information about the full list of supported variables visit https://vertx.io/docs/vertx-pg-client/java/#_environment_variables[vertx-pg-client]

Reviewer: ```suggestion
You can now configure the connection with PostgreSQL using environment variables. For example, you can use `PGUSER` and `PGPASSWORD` to specify the username and password. For more information about the full list of supported variables, see https://vertx.io/docs/vertx-pg-client/java/#_environment_variables[vertx-pg-client]
```
